library/cluster-config/wait-for-configuration.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+. $(dirname $0)/../store/store.sh
+
+wait_for_configure() {    
+    first_node=$(jq -r '.nodes_setting[0].management_interface.fqdn_name' /home/abhijith/workspace/nbfs/dr-setup/config/3.1/variables-${CLUSTER}.json)
+    if [ "${first_node}" == "" ]; then
+        echo "Failed to get the first node fqdn."
+        exit 1
+    fi
+    first_node_url=${first_node}:8443
+    
+    node_auth_token=$(get_auth_token ${first_node})
+    node_cookie_file=$(get_cookie_file_path ${first_node})
+
+    wait_time=60
+    while [ ${wait_time} ] >0; do
```

Comment:
This tests whether the variable is empty, and then writes the (empty) output of the test to a file named _0_. I think you wanted to test the numeric value instead.

```suggestion
    while ((wait_time > 0)); do
```

On the other hand, I don't see where `$wait_time` changes in this loop, so the condition doesn't make sense anyway.